### PR TITLE
Singularity lexer text analysis

### PR DESCRIPTION
**Pull request by gandarez**
This PR ports pygments Singularity text analysis to chroma/go. Original code can be found at: https://github.com/pygments/pygments/blob/master/pygments/lexers/configs.py#L974

### CODE REVIEW

**Comment by gandarez** [comment]
This [PR](https://github.com/wakatime/chroma/pull/34) fixes the build

File: lexers/s/singularity.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		MimeTypes: []string{},
+	},
+	Rules{
+		"root": {},
+	},
+).SetAnalyser(func(text string) float32 {
+	// This is a quite simple script file, but there are a few keywords
+	// which seem unique to this language.
+
+	var result float32 = 0
```
Comment (dron22) [inline lexers/s/singularity.go]: You can make it more brief and we do not need the blank line (I'm playing the linter here :wink: )
```suggestion
	var result float32
```

**Review by dron22 — APPROVED** [review]
Only one small remark

File: lexers/s/singularity.go (hunk 1 above)
Comment (gandarez) [reply lexers/s/singularity.go]: We need to apply the same behaviour to the others also. Will do a separate PR for that.

**Pull request merged**